src/Orchard.Cms.Web/Modules/Orchard.OpenId/Module.txt

```diff
@@ -0,0 +1,10 @@
+﻿Name: OpenID
+AntiForgery: enabled
+Author: The Orchard Team
+Website: http://orchardproject.net
+Version: 2.0.x
+OrchardVersion: 2.0.x
+Description: The OpenID module enables authentication of external apps through OpenID Connect.
+FeatureDescription: OpenID Connect apps management and authentication.
```

Comment:
A possible split would have been one feature per supported flow. Instead of enabling flows from OpenId Settings they are enabled as modules. Initially it could seem a good idea because it will avoid the need of restarting the site after changing supported flows. Problem is that if you change also other settings like Authority, Audiences or Testing Mode, the reboot is still needed. So we don't solve the issue.